src/components/Form.js

```diff
@@ -0,0 +1,41 @@
+import { useState } from "react";
+
+
+const Form = ( { addTask } ) => {
+    const [task, setTask] = useState("")
+
+    const handleFormSubmit = (e) => {
+        e.preventDefault(); //prevent the default refresh after submit
+        addTask({
+            name: task,
+            checked: false,
+            id: Date.now() //timestemp for random key
+        })
+        setTask("");
+    }
+
+    return <form className="todo" onSubmit={handleFormSubmit}>
```

Comment:
I wouldn't use a form element just for one input, useState of the input value is enough

Reply:
> I wouldn't use a form element just for one input, useState of the input value is enough

I would like that you explain about it more, I used to develop in PHP that is base on refresh and in order to execute an input method its needed to be on "Form" only.